### PR TITLE
Check title or description

### PR DESCRIPTION
**Pull request by sarangan12**
Related to https://github.com/Azure/autorest/issues/1767

### CODE REVIEW

File: src/core/AutoRest.Core/Properties/Resources.resx [hunk 1]
```diff
@@ -268,6 +268,9 @@ Modelers:
   <data name="MissingDescription" xml:space="preserve">
     <value>{0} lacks 'description' property. Consider adding a 'description' element. Accurate description is essential for maintaining reference documentation.</value>
   </data>
+  <data name="MissingTitleDescription" xml:space="preserve">
+    <value>{0} lacks 'description' and 'title' property. Consider adding a 'description'/'title' element. Accurate description/title is essential for maintaining reference documentation.</value>
```
Comment (dsgouda) [inline src/core/AutoRest.Core/Properties/Resources.resx]: Do you mean ```Description or a title``` property

File: src/modeler/AutoRest.Swagger/Validation/DescriptionAndTitleMissing.cs [hunk 2]
```diff
@@ -5,24 +5,33 @@
 using AutoRest.Core.Logging;
 using AutoRest.Swagger.Validation.Core;
 using AutoRest.Swagger.Model;
+using AutoRest.Core.Properties;
 
 namespace AutoRest.Swagger.Validation
 {
-    public class DescriptionMissing : DescriptionRequired<Dictionary<string, Schema>>
+    public class DescriptionAndTitleMissing : DescriptionRequired<Dictionary<string, Schema>>
```
Comment (dsgouda) [inline src/modeler/AutoRest.Swagger/Validation/DescriptionAndTitleMissing.cs]: Shouldn't we extend all our description-related validation rules to the title nodes as well.

File: src/core/AutoRest.Core/Properties/Resources.resx (hunk 1 above)
Comment (sarangan12) [inline src/core/AutoRest.Core/Properties/Resources.resx]: It is 'and'. If atleast one of them exist, we do not raise this error

File: src/modeler/AutoRest.Swagger/Validation/DescriptionAndTitleMissing.cs (hunk 2 above)
Comment (sarangan12) [inline src/modeler/AutoRest.Swagger/Validation/DescriptionAndTitleMissing.cs]: The Only other rule is Descriptive Description. Do you mean that? If not, Please clarify what do you mean by extending?

Comment (dsgouda) [inline src/modeler/AutoRest.Swagger/Validation/DescriptionAndTitleMissing.cs]: I meant this change should cover other description related scenarios

Comment (sarangan12) [inline src/modeler/AutoRest.Swagger/Validation/DescriptionAndTitleMissing.cs]: Already checked it. This seems to be the only place

**Pull request merged**